### ISSUE TITLE
Expose UUINavWidget.bParentRemoved to blueprints

### ISSUE THREAD

**Issue by Levitikon217** [goncasmage1/UINavigation#29]
It would be really cool if you could expose bParentRemoved to blueprints. Spent a minute trying to figure out why parents were closed when adding child. Tracked it to this bool but I had to enable in c++.

**Comment by goncasmage1** [comment]
What do you mean by adding a child?

A sábado, 21/09/2019, 20:42, Levitikon <notifications@github.com> escreveu:

**Comment by Levitikon217** [comment]
When creating a new UINavWidget (child) from within a UINavWidget (parent) and setting the parent, like so:
![image](https://user-images.githubusercontent.com/1824102/65387131-2582f500-dd12-11e9-9c4b-38ca29c44768.png)

Will cause the parent to disappear. I can imagine this would be expected default behavior, right? If I'm in a main-menu (parent) and open a sub-menu (child), the main-menu should disappear until I come back. Makes sense. 

However, I came across a scenario when I want the parent to remain and have a transparent background child show above. A sort of layered menu thing. This is a case where I don't want the parent to disappear.

Fortunately, I came across the key in the ctor of the UINavWidget. There is an if-condition 
```
	/*
	If this widget was added through a parent widget and should remove it from the viewport,
	remove that widget from viewport
	*/
	if (ParentWidget != nullptr && ParentWidget->IsInViewport() && bParentRemoved)
	{
		ParentWidget->RemoveFromParent();
	}
```

All I really need is to toggle bParentRemoved to false and the parent won't disappear. The problem is, bParentRemoved isn't exposed to blueprints to easily toggle it. Easy enough though, I just exposed it to blueprints and configured my requirements accordingly and everything works great.
```
	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = UINavButton)
	bool bParentRemoved = true;
```

Just thought this might be a nice thing for others to leverage in-case they run into similar situation.
